src/Documents/Document.ts

```diff
@@ -0,0 +1,15 @@
+import {IDocument} from './IDocument';
+
+export class Document implements IDocument {
+  public entityName(): string {
+      return this.constructor.name;
+  }
+
```

Comment:
@a-serdukov-mobilunity please adjust the code based on our above comments. If you have any questions concerning this change let's talk tomorrow.